utils/rad

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env radicle
```

Comment:
Can we move this to the `bin` directory? I’d also like to avoid overgeneralization and call the script `rad-issues`. This way we don’t force ourselves yet to build a unified CLI.

Reply:
> Can we move this to *the* bin directory

Do you mean to a *new* `bin` directory? There currently isn't one.